src/Chapter06/Listing06.20.ProvidingPropertyValidation.cs

```diff
@@ -24,38 +24,32 @@ public string LastName
             get => _LastName;
             set
             {
-                #region HIGHLIGHT
+                // #region EXCLUDE
+                #if !NET7_0_OR_GREATER
+                _LastName = value;
+                value=value?.Trim()!; // Remove surrounding whitespace.
```

Comment:
Do we actually need the null checks here? `value` is non-nullable.

nit: Whitespace
```suggestion
                value = value.Trim(); // Remove surrounding whitespace.
```